server/plugin/command.go

```diff
@@ -34,14 +72,39 @@ func (p *MatterpollPlugin) ExecuteCommand(c *plugin.Context, args *model.Command
 	siteURL := *p.ServerConfig.ServiceSettings.SiteURL
 	configuration := p.getConfiguration()
 
+	user, appErr := p.API.GetUser(creatorID)
+	if appErr != nil {
+		return getCommandResponse(model.COMMAND_RESPONSE_TYPE_EPHEMERAL, commandErrorGeneric.Other, siteURL, nil), nil
+	}
+	localizerUser := i18n.NewLocalizer(p.bundle, user.Locale)
+	localizerPublic := i18n.NewLocalizer(p.bundle, *p.ServerConfig.LocalizationSettings.DefaultServerLocale)
+
+	defaultYes := localizerPublic.MustLocalize(&i18n.LocalizeConfig{DefaultMessage: commandDefaultYes})
+	defaultNo := localizerPublic.MustLocalize(&i18n.LocalizeConfig{DefaultMessage: commandDefaultNo})
+
 	q, o, s := utils.ParseInput(args.Command, configuration.Trigger)
 	if q == "" || q == "help" {
-		msg := fmt.Sprintf(commandHelpTextFormat, configuration.Trigger)
+		msg := localizerUser.MustLocalize(&i18n.LocalizeConfig{
+			DefaultMessage: commandHelpTextSimple,
+			TemplateData:   map[string]interface{}{"Trigger": configuration.Trigger, "Yes": defaultYes, "No": defaultNo},
+		}) + "\n"
+		msg += localizerUser.MustLocalize(&i18n.LocalizeConfig{
+			DefaultMessage: commandHelpTextOptions,
+			TemplateData:   map[string]interface{}{"Trigger": configuration.Trigger},
+		}) + "\n"
+		msg += localizerUser.MustLocalize(&i18n.LocalizeConfig{
+			DefaultMessage: commandHelpTextPollSettingIntroduction,
+			TemplateData:   map[string]interface{}{"Trigger": configuration.Trigger},
+		}) + "\n"
+		msg += "- `--anonymous`: " + localizerUser.MustLocalize(&i18n.LocalizeConfig{DefaultMessage: commandHelpTextPollSettingAnonymous}) + "\n"
+		msg += "- `--progress`: " + localizerUser.MustLocalize(&i18n.LocalizeConfig{DefaultMessage: commandHelpTextPollSettingProgress}) + "\n"
+		msg += "- `--public-add-option`: " + localizerUser.MustLocalize(&i18n.LocalizeConfig{DefaultMessage: commandHelpTextPollSettingPublicAddOption})
```

Comment:
Any ideas on how to make this prettier?

Reply:
I'm fine with messages to be translated per line. 2/5

i.e.:
```
msg += localizerUser.MustLocalize(&i18n.LocalizeConfig{
    DefaultMessage: commandHelpTextPollSettingAnonymous,
    TemplateData:   map[string]interface{}{"AnonymousSetting": "--anonymous"},
})
msg += localizerUser.MustLocalize(&i18n.LocalizeConfig{
    DefaultMessage: commandHelpTextPollSettingProgress,
    TemplateData:   map[string]interface{}{"ProgressSetting": "--pregress"},
})
msg += localizerUser.MustLocalize(&i18n.LocalizeConfig{
    DefaultMessage: commandHelpTextPollSettingPublicAddOption,
    TemplateData:   map[string]interface{}{"PublicAddOptionSetting": "--public-add-option"},
})
```

```
commandHelpTextPollSettingAnonymous = &i18n.Message{
    ID:    "command.help.text.pollSetting.anonymous",
	Other: "- {{.AnonymousSetting}}: Don't show who voted for what\n",
}
...
```

Reply:
What is the great difference between your proposal and what I did besides moving the setting name inside the translation?